chapter4/e2-recursion.js

```diff
@@ -0,0 +1,6 @@
+function isEven(n) {
+  if (n < 0) return isEven(-n);
+  if (n == 0) return "Even";
+  if (n == 1) return "Odd";
+  return isEven(n - 2);
+}
```

Comment:
You should always add test cases to these. Nothing crazy, just a 1, 0, negative, big positive, and maybe a 2.